### PR TITLE
Capitalization in Conditions

### PR DESCRIPTION
**Pull request by rosebrallier**
-Capitalized uncapitalized Conditions per Issue #792 

Then I couldn't help letting the scope creep a wee bit:
-Fixed some spelling, grammar, and awkward wording/sen struct
-Continued my crusade to consistently notate Skill checks
-Look, I don't care if "Character" is in our Glossary. It's weird to capitalize a wholly universal term that we use in exactly the same way as every other type of media in the world /endrant
-Changed d10 to 1d10 on the philosophy that d10 is the dice type, 1d10 is a) clear that that we're rolling one die and b) consistent with how we notate rolling multiple dice
-Slightly buffed Pinned to force a Pinned character to hunker, rather than implying that becoming Pinned causes a character to hunker as a free action
-Fixed an instance of the nonexistent Fort save
-Swapped "drunk" with "Intoxicated" to allow room to include non-alcoholic intoxicating substances in a future patch to Buzzed and Hammered
-Changed the result of blacking out from Downed to Stunned b/c I don't think we want someone to insta-die b/c they got punched while blacked out and died b/c they were Downed and so had 0 hp. Gave the Stun a 2 hr duration b/c that's how long the other intoxication effects last. NOTE: Need to specify what happens when the Stun wears off. Also, I really should get around to writing Unconscious, b/c it can have the added effect of the character physically collapsing.
-Explicitly added Afraid as referencing the same Condition as Fear. Idk how to to incorporate that into Greater Fear, but that's a problem for Future Me. (Greater Afraid. -snort-)

Please respond specifically to the discussion points/major changes

### CODE REVIEW

File: 01_05_Conditions.txt [hunk 1]
```diff
@@ -9,100 +9,99 @@ anatomy that contian liquids vital to their survival. This can happen to both
 biologic targets, who usually leak blood, and to cybernetic targets, who can 
 leak coolant and lubricant.
 
-    A Character that is bleeding takes d10/2+1 (drop the remainder) stackable 
-damage once per round at the beginning of their turn. The bleeding can be 
-staunched with Medicine Skill check. The Medicine (Dex) DC to stop varies; the
-cause of the bleed effect should list it in the effect description.
+    A character that is Bleeding takes d10/2 + 1 (drop the remainder) stackable
```
Comment (Turtlelord26) [inline 01_05_Conditions.txt]: 1d10 / 2 + 1

File: 01_05_Conditions.txt [hunk 2]
```diff
@@ -9,100 +9,99 @@ anatomy that contian liquids vital to their survival. This can happen to both
...
 universe.
 
-    A Character that has the Fear Condition may take no actions, except moving 
-away from the source of fear, or taking an action to roll to defeat fear. DC 50, 
-Will Save.
+    A character that is Afraid may take no actions, except moving away from
+the source of fear, or taking an action to roll a DC 50 Will Save to defeat Fear.
 
-== Fear, Greater ==
+== Fear, Greater (Afraid) ==
```
Comment (Turtlelord26) [inline 01_05_Conditions.txt:74]: Deeply Afraid?
Actually, as it stands it's consistent with Buzzed/Hammered (Intoxicated)

File: 01_05_Conditions.txt [hunk 3]
```diff
@@ -9,100 +9,99 @@ anatomy that contian liquids vital to their survival. This can happen to both
...
     Being Lonely is a condition most commonly contracted from having 1 Charisma
 and failing to successfully socialize, or by trying and failing to socialize 
-with an ally who has 1 Charisma. A Lonely Character hasn't met Mazlow's 
+with an ally who has 1 Charisma. A Lonely character hasn't met Mazlow's 
 Hierarchy of needs, and this affects their performance. 
 
-    A Lonely Character moves 1m slower every two Actions (round down) and takes 
-a -5 to skill rolls. 
+    A Lonely character moves 1m slower every two Actions (round down) and takes 
+a -5 to Skill checks.
```
Comment (Turtlelord26) [inline 01_05_Conditions.txt]: Do we need to capitalize skill?

File: 01_05_Conditions.txt [hunk 4]
```diff
@@ -9,100 +9,99 @@ anatomy that contian liquids vital to their survival. This can happen to both
...
-with an ally who has 1 Charisma. A Lonely Character hasn't met Mazlow's 
+with an ally who has 1 Charisma. A Lonely character hasn't met Mazlow's 
 Hierarchy of needs, and this affects their performance. 
 
-    A Lonely Character moves 1m slower every two Actions (round down) and takes 
-a -5 to skill rolls. 
+    A Lonely character moves 1m slower every two Actions (round down) and takes 
+a -5 to Skill checks. 
 
 == Necrosis ==
```
Comment (Turtlelord26) [inline 01_05_Conditions.txt:95]: This is a definite rabbit hole, but we should totally make a Corrosion condition that does this for robotic species.

File: 01_05_Conditions.txt [hunk 5]
```diff
@@ -9,100 +9,99 @@ anatomy that contian liquids vital to their survival. This can happen to both
...
     Some burning chemicals, like white phosphorous or most napalms, contain the 
 oxygen that they need to burn and aren't extinguished with water or rolling 
 around. These sources of the Burning Condition will explicitly state that they 
-can't be put out with normal means and will need to be neutralized by other 
+can't be extinquished via normal means and will need to be neutralized by other 
 means, such as the Doctor's Purge ability.
 
 == Downed ==
 
-    Downed is a state that a character enteres when their health has fallen
```
Comment (bleehu) [inline 01_05_Conditions.txt:40]: ew. British speak.

File: 01_05_Conditions.txt [hunk 6]
```diff
@@ -112,62 +111,61 @@ biofoam.
 fire or a continuous stream of bullets must overcome the psychological instinct
 to stay put and not get shot.
 
-    A target that is pinned is hunkered and cannot move for 2 full Rounds or
```
Comment (bleehu) [inline 01_05_Conditions.txt:115]: Should we specify that this hunker is an Immediate Reaction that costs Actions on their turn?

**Review by Turtlelord26 — APPROVED** [review]
### Gigantic Grammar Glob Greatly Galvanizes Glossary Glee!

> -Changed d10 to 1d10 on the philosophy that d10 is the dice type, 1d10 is a) clear that that we're rolling one die and b) consistent with how we notate rolling multiple dice

Yes. Good.

> -Explicitly added Afraid as referencing the same Condition as Fear.

That will help rules writers.

> -Slightly buffed Pinned to force a Pinned character to hunker, rather than implying that becoming Pinned causes a character to hunker as a free action

I'll greenlight this as the kind of change that will be pretty apparent in testing if it feels too ick. It's also quite easy to change the ordering of the hunker and save in the text should we need to.

> Also, I really should get around to writing Unconscious, b/c it can have the added effect of the character physically collapsing.

🐢

File: 01_05_Conditions.txt [hunk 7]
```diff
@@ -112,62 +111,61 @@ biofoam.
...
     Many drinks in the galaxy contain a substance called alcohol. When consuming 
-these drinks, the effect should list a FORT save to avoid increasing your level 
-of intoxication. The first level of intoxication is Buzzed. 
+these drinks, the effect should list a Shock Save to avoid increasing the character's
+level of intoxication. The first level of intoxication is Buzzed. 
 
-    While buzzed, a Character gains a -1 to accuracy with any attack and a 
-temporary -1 to Perception. At the same time, the Character gets a +1 to 
-Charisma and Luck. You also take a penalty of -20 to skill checks involving 
-driving, flying or operating heavy machinery. If the Character fails a Fort save
```
Comment (bleehu) [inline 01_05_Conditions.txt:153]: Should we use one of those God-awful Oxford comas here?

**Review by bleehu — APPROVED** [review]
This is a monumental effort! Thanks doing this R! 
There's some discussion to be had about the design of what happens if your character goes on a bender and drinks until they're in danger, but that's beyond the scope of this PR, and I accept that you can kick a person who's passed out drunk without killing them.

File: 01_05_Conditions.txt (hunk 2 above)
Comment (rosebrallier) [reply 01_05_Conditions.txt:74]: That's why I stuck it in there :)

**Pull request merged**